.pre-commit-config.yaml

```diff
@@ -32,6 +32,7 @@ repos:
   rev: v2.9.0
   hooks:
     - id: pip-audit
+      args: ["--ignore-vuln", "GHSA-4xh5-x5gv-qwph"]
```

Comment:
is there a reason we are ignoring this vuln?

Reply:
i only recommended this because ive encountered this and its intermittent.
It depends on what machine I'm on.
It may just not be configure to use pixi's pip, and is using the one on your default env.